### PR TITLE
nautilus and deps updates

### PR DESCRIPTION
**Pull request by satmandu**
- Gnome Nautilus & dependencies

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/libassuan.rb [hunk 1]
```diff
@@ -3,33 +3,38 @@
...
   depends_on 'libgpgerror'
 
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
+    system '[ -x configure ] || NOCONFIGURE=1 ./autogen.sh'
+    system 'filefix'
+    system "env CFLAGS='-flto=auto' \
+      CXXFLAGS='-pipe -flto=auto' \
+      LDFLAGS='-flto=auto' \
+      ./configure"
```
Comment (uberhacker) [inline packages/libassuan.rb:33]: No `#{CREW_OPTIONS}`?

File: packages/libgpgerror.rb [hunk 2]
```diff
@@ -3,30 +3,32 @@
...
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-maintainer-mode'
+    system '[ -x configure ] || NOCONFIGURE=1 ./autogen.sh'
+    system "env CFLAGS='-flto=auto' \
+      CXXFLAGS='-pipe -flto=auto' \
+      LDFLAGS='-flto=auto' \
+      ./configure \
+      --disable-maintainer-mode"
```
Comment (uberhacker) [inline packages/libgpgerror.rb:31]: No `#{CREW_OPTIONS}`?

File: packages/smbclient.rb [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  depends_on 'libunwind'
+  depends_on 'liburing' => ':build'
+  depends_on 'linux_pam'
+  depends_on 'lmdb' => ':build'
+  depends_on 'perl_parse_yapp' => ':build'
+  depends_on 'popt'
+  depends_on 'popt' => ':build'
+  depends_on 'talloc'
+  depends_on 'tdb'
+  depends_on 'tevent'
```
Comment (uberhacker) [inline packages/smbclient.rb:56]: No need to include both runtime and build dependencies.  Just include the runtime dependency.

Comment (satmandu) [reply packages/smbclient.rb:56]: That's quite reasonable.

Comment (uberhacker) [reply packages/smbclient.rb:56]: Also, symbols are not quoted.

File: packages/nautilus.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  depends_on 'gnome_desktop'
+  depends_on 'gvfs'
+  depends_on 'dconf'
+  depends_on 'libhandy'
+  depends_on 'libportal'
+  depends_on 'tracker3'
+  depends_on 'tracker3_miners'
+  depends_on 'gobject_introspection' => ':build'
+  depends_on 'gtk_doc' => ':build'
+  depends_on 'appstream_glib' => ':build'
```
Comment (uberhacker) [inline packages/nautilus.rb:40]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/libgrss.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  binary_sha256({
+    aarch64: '3ec7ce667887023dd6766c726f8c054daafe58fe9003f4ac52203f73f956a743',
+     armv7l: '3ec7ce667887023dd6766c726f8c054daafe58fe9003f4ac52203f73f956a743',
+       i686: '95b2d74dd324c1616d29e7d0b6794dcd52c231d486a0fc28271fa0ba17367aae',
+     x86_64: 'cf9f47cce02e43332117ce05f76c0c61aa7bf35154261e6d175fc4711024a3eb'
+  })
+
+  depends_on 'libsoup'
+  depends_on 'gtk_doc' => ':build'
+  depends_on 'gobject_introspection' => ':build'
```
Comment (uberhacker) [inline packages/libgrss.rb:30]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/ldb.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  })
+
+  depends_on 'talloc'
+  depends_on 'tevent'
+  depends_on 'tdb'
+  depends_on 'popt'
+  depends_on 'lmdb'
+  depends_on 'cmocka' => ':build'
+  depends_on 'docbook_xsl' => ':build'
+  depends_on 'tevent' => ':build'
```
Comment (uberhacker) [inline packages/ldb.rb:35]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/libimobiledevice.rb [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  binary_sha256({
+    aarch64: '7a8a73892ac722226e44a17c810900213aa6c5e66c26ad23b8d30b379d5b7dae',
+     armv7l: '7a8a73892ac722226e44a17c810900213aa6c5e66c26ad23b8d30b379d5b7dae',
+       i686: '2580d6ecf627dce6a4dd6018ef3245e7352a95ac2fcd586627188379dc4489fa',
+     x86_64: '11462fab4814036224fb3078792bad480627ead538309afcd93abfbb85121d65'
+  })
+
+  depends_on 'libusbmuxd'
+  depends_on 'libplist' => ':build'
+  depends_on 'autoconf_archive' => ':build'
```
Comment (uberhacker) [inline packages/libimobiledevice.rb:30]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/talloc.rb [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+  })
+  binary_sha256({
+    aarch64: '34e7e59fdd8ab619eb0a24aec3011093653399502038b7071395006dcf6e3b08',
+     armv7l: '34e7e59fdd8ab619eb0a24aec3011093653399502038b7071395006dcf6e3b08',
+       i686: 'f2b0b683234c60440ac6c53773aa8f38297ac8548c1a509773fa5bae836fbf47',
+     x86_64: '0b22a9f1ce28259ba8d4fd046d00677c6b2590f2a0951a004d1f9d6173e592b6'
+  })
+
+  depends_on 'libxcrypt'
+  depends_on 'docbook_xsl' => ':build'
```
Comment (uberhacker) [inline packages/talloc.rb:29]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/tracker3.rb [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  })
+
+  depends_on 'asciidoc'
+  depends_on 'docbook_xml'
+  depends_on 'gobject_introspection' => ':build'
+  depends_on 'gtk_doc' => ':build'
+  depends_on 'libsoup'
+  depends_on 'libstemmer'
+  depends_on 'util_linux'
+  depends_on 'vala' => ':build'
```
Comment (uberhacker) [inline packages/tracker3.rb:35]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/tracker3_miners.rb [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+  depends_on 'tracker3'
+  depends_on 'gst_plugins_base'
+  depends_on 'exempi'
+  depends_on 'libexif'
+  depends_on 'libcue'
+  depends_on 'libgrss'
+  depends_on 'libgsf'
+  depends_on 'vala' => ':build'
+  depends_on 'asciidoc' => ':build'
```
Comment (uberhacker) [inline packages/tracker3_miners.rb:36]: Symbols are not quoted.  Use `:build`, not `':build'`.

File: packages/libassuan.rb (hunk 1 above)
Comment (satmandu) [reply packages/libassuan.rb]: Fixed

File: packages/libgpgerror.rb (hunk 2 above)
Comment (satmandu) [reply packages/libgpgerror.rb:31]: fixed.

**Comment by satmandu** [comment]
requested changes made.

**Pull request merged**